### PR TITLE
feat(api): Add PipelineTaskFinalStatus to pipeline_spec.proto

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**
`PipelineTaskFinalStatus` is the value format for `TaskFinalStatus`, which is used by Exit handler.
https://github.com/kubeflow/pipelines/blob/97b9bdb530adaf824ff9aee14ff1a8ac70eb063e/api/v2alpha1/pipeline_spec.proto#L260

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: api/v2alpha1/google/rpc/status.proto [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
```
Comment (chensun) [inline api/v2alpha1/google/rpc/status.proto:1]: This file isn't available via protoc's standard include path. Having a local copy (forked from https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto) seems to be the simplest approach.
But let me know if theres's a better solution.

**Review by hilcj — APPROVED** [review]

File: api/v2alpha1/google/rpc/status.proto (hunk 1 above)
Comment (neuromage) [reply api/v2alpha1/google/rpc/status.proto:1]: You could potentially use a Makefile to copy during compilation with protoc. That way, you can specify a version. Copying locally seems ok too. @Bobgy may have some ideas on this as well.

Comment (chensun) [reply api/v2alpha1/google/rpc/status.proto:1]: Missed this comment before merging. Yes, I can do it this way as well.

**Pull request merged**